views/home/viewreference.ejs

```diff
@@ -42,13 +42,23 @@
         <div class="col-md-6">
           <div class="panel panel-default">
             <div class="panel-heading">
-              <strong>Notes</strong>
+              <strong>Public Notes</strong>
             </div>
             <div class="panel-body">
               <md ng-model="selectedRef.notes"></md>
             </div>
           </div>
         </div>
+        <div ng-if="selectedRef.privatenotes !== null" class="col-md-6">
```

Comment:
Should users be able to see private notes when they view their own public profile? At the moment they can, but it might mislead users into thinking that others can also see their private notes.

If we decide to hide the notes, it would be easiest to just do a clientside fix by adding another condition to this ng-if.